src/app/modules/startup_v2/module.nim

```diff
@@ -0,0 +1,49 @@
+import NimQml, chronicles, json
+
+import io_interface
+import view, controller
+
+import app/global/global_singleton
+import app/core/eventemitter
+
+export io_interface
+
+logScope:
+  topics = "startup-v2-module"
+
+type
+  Module*[T: io_interface.DelegateInterface] = ref object of io_interface.AccessInterface
+    delegate: T
+    view: View
+    viewVariant: QVariant
+    controller: Controller
+
+proc newModule*[T](delegate: T, events: EventEmitter): Module[T] =
+  result = Module[T]()
+  result.delegate = delegate
+  result.view = view.newView(result)
+  result.viewVariant = newQVariant(result.view)
+  result.controller = controller.newController(result, events)
+
+{.push warning[Deprecated]: off.}
+
+method delete*[T](self: Module[T]) =
+  self.view.delete
+  self.viewVariant.delete
+  self.controller.delete
+
+method onAppLoaded*[T](self: Module[T]) =
+  singletonInstance.engine.setRootContextProperty("startupV2Module", newQVariant())
```

Comment:
In the OnboardingStore, we have sth like:
```qml
QtObject {
    readonly property QtObject d: StatusQUtils.QObject {
        id: d
        readonly property var onboardingModuleInst: onboardingModule
    }
```
we might want to add the `v2` there as well, but not "startup" module I think